assembl/static2/js/app/graphql/IdeaWithPosts.graphql

```diff
@@ -8,7 +8,7 @@ query IdeaWithPosts($lang: String!, $id: ID!) {
       posts(first: 20) {
         edges {
           node {
-            ... on Post {
+            ... on AssemblPost {
```

Comment:
same here, put back Post in the query or you won't get the ImportedPost instances.